si-id-epu50/si_id_epu50/ecodrive.py

```diff
@@ -2,96 +2,87 @@
 
 import logging
 logger = logging.getLogger(__name__)
-import logging.handlers as handlers
 import socket
 import threading
 import time
 
 from . import constants as _cte
 from .utils import *
```

Comment:
try not to import this way. Maybe `from . import utils` or `import .utils as _utils` would be better.